packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -106,7 +106,15 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
 
   public componentDidMount(): void {
     const isChartEmpty =
-      this.state.emptyChart ||
+      this._points.length === 0 ||
```

Comment:
Is this issue only for vertical bar charts and not others?

Reply:
as discussed offline, raising a separate PR for all other charts